packages/api-gateway/src/drivers/http/adapters/admin_panel/handlers.js

```diff
@@ -1,11 +1,35 @@
 async function sayHello(req, reply) {
-  const result = await this.adminPanelService.sayHello();
+
+	const result = await this.adminPanelService.sayHello();
```

Comment:
Can you replace the tab with 2 spaces? it is throwing eslint warnings
```suggestion
  const result = await this.adminPanelService.sayHello();
```